### PR TITLE
miniupnpc: add v2.2.5, do not run upnpDiscover() in test_package

### PR DESCRIPTION
**Pull request by valgur**

### CODE REVIEW

File: recipes/miniupnpc/all/test_package/test_package.c [hunk 1]
```diff
@@ -7,6 +7,8 @@ int main(void) {
     int ipv6 = 0;
     unsigned char ttl = 2;
     int error = 0;
-    struct UPNPDev *devlist = upnpDiscover(2000, multicastif, minissdpdpath, localport, ipv6, ttl, &error);
+    if (0) {
```
Comment (RubenRBS) [inline recipes/miniupnpc/all/test_package/test_package.c:10]: Any reasoning for disabling this, and doing it this way?

**Review by RubenRBS — COMMENTED** [review]
Thanks!

File: recipes/miniupnpc/all/test_package/test_package.c (hunk 1 above)
Comment (valgur) [reply recipes/miniupnpc/all/test_package/test_package.c:10]: It probes the local network for UPnP support. It takes a couple of seconds to complete and can look a bit suspicious in terms of network security. For test_package it's sufficient to see that it links correctly.

Comment (RubenRBS) [reply recipes/miniupnpc/all/test_package/test_package.c:10]: Yup! Good idea. Why not remove the whole line though?

Comment (valgur) [reply recipes/miniupnpc/all/test_package/test_package.c:10]: Just to confirm that linking works correctly for the library.

Comment (RubenRBS) [reply recipes/miniupnpc/all/test_package/test_package.c:10]: Ahhh, I see I see, sorry I didnt catch what you meant in your first comment :) thanks as always!

**Review by RubenRBS — APPROVED** [review]

**Review by ericLemanissier — APPROVED** [review]

**Pull request merged**